test/e2e/Tests/Tests/OrchestrationQueryTests.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Microsoft.Azure.Durable.Tests.DotnetIsolatedE2E;
 
-[Collection(Constants.FunctionAppCollectionName)]
+[Collection(Constants.FunctionAppCollectionSequentialName)]
```

Comment:
Just to confirm -- this attribute ensures the test runs after the other tests?

Reply:
Correct - per XUnit documentation, tests that are part of a collection initialized with the parameter DisableParallelization = true will run after all parallelized collections. Each test in these collections will be run independently. 
Testing locally, I saw our current suite complete approx. 20s slower with this change - something to keep in mind when writing new tests is only mark them sequential if it is necessary.